### PR TITLE
Set up path for toolset.

### PR DESCRIPTION
**Pull request by jashook**
Set up the path for the arm64 toolset.

### CODE REVIEW

**Comment by jashook** [comment]
@mmitche PTAL

**Comment by mmitche** [comment]
LGTM

**Comment by jashook** [comment]
@gkhanna79 PTAL

File: build.cmd [hunk 1]
```diff
@@ -139,14 +142,28 @@ if not exist %_msbuildexe% echo Error: Could not find MSBuild.exe.  Please see h
...
 echo Commencing build of native components for %__BuildOS%.%__BuildArch%.%__BuildType%
 echo.
 
 rem arm64 builds currently use private toolset which has not been released yet
-if /i "%__BuildArch%" =="arm64" goto GenVSSolution
+if /i "%__BuildArch%" == "arm64"  (
+  REM TODO, remove once the toolset is open.
+
+  if /i "%__ToolsetDir%" == "" (
+    echo "A toolset directory is required for the Arm64 Windows build."
```
Comment (gkhanna79) [inline build.cmd]: NIT: "Windows Arm64"

File: build.cmd [hunk 2]
```diff
@@ -139,14 +142,28 @@ if not exist %_msbuildexe% echo Error: Could not find MSBuild.exe.  Please see h
...
+    exit /b 1
+  )
+)
+
+REM TODO, remove once the toolset is open.
+if /i "%__BuildArch%" == "arm64" set PATH=%PATH%;%__ToolsetDir%\cpp\bin
```
Comment (gkhanna79) [inline build.cmd]: Can you please convert this to a single if that sets all the required variables?

Comment (mmitche) [inline build.cmd]: @gkhanna79 That doesn't work as expected.  The expansion rules for paren blocks cause problems with the path setup (even with the delayed expansion).  vcvars does what @jashook is doing here.

**Comment by jashook** [comment]
Nice catch, should be updated to set the path only on arm64 windows. Also as Matt said, we followed the vsvars32 approach to set the path.

**Comment by jashook** [comment]
Update, work in progress do not merge.

**Comment by mmitche** [comment]
LGTM

**Pull request merged**